### ISSUE TITLE
Document origins of "pdf" versions of pre 5.5 specifications

### ISSUE THREAD

**Issue by chronoplexsoftware** [FamilySearch/GEDCOM#52]
Copies of GEDCOM 3.0, 4.0, 5.0, 5.3, 5.4 have all surfaced online at some point over the last 6 or 7 years.  However, these have been presented in a range of inaccessible or obsolete formats which were often unsuitable for sharing or searching.

Since 2015, we have converted GEDCOM 3.0, 4.0, 5.0, 5.3, and 5.4 to searchable pdfs (hosted [here](https://chronoplexsoftware.com/gedcomvalidator/gedcom/)), some of which are now being hosted directly on [gedcom.io/specs](https://gedcom.io/specs/).  We would like to ensure that the original sources and conversion process is documented and retained.

@luther Specifically for GEDCOM 3.0 and 4.0, several minor transcription errors have been corrected since these specs were first published as text pdfs. It is suggested that you pull down current copies of those transcriptions from [chronoplexsoftware.com](https://chronoplexsoftware.com/gedcomvalidator/gedcom/) to replace those currently published on gedcom.io.  The current one dates from 2019 and 2018 respectively.

**GEDCOM 3.0** is only known to exist as original paper copies and scans thereof:  
http://www.beholdgenealogy.com/GEDCOM3.0.pdf

That scan was transcribed using OCR and manually corrected to produce the following pdf first made available in Jul 2016:
https://chronoplexsoftware.com/gedcomvalidator/gedcom/gedcom-3.0.pdf

**GEDCOM 4.0** is only known to exist as original paper copies and scans thereof.  
https://web.archive.org/web/20160318183421/http://www.genpluswin.de/gedcom/Gedcom_4.pdf

That scan was transcribed using OCR and manually corrected to produce the following pdf first made available in Feb 2015:
https://chronoplexsoftware.com/gedcomvalidator/gedcom/gedcom-4.0.pdf

**GEDCOM 5.0** exists in original form as a DOS text version with a txt extension.  
https://www.beholdgenealogy.com/Gedcom.standard.5.txt

This was manually reformatted in Microsoft Word and converted to pdf:
https://chronoplexsoftware.com/gedcomvalidator/gedcom/gedcom-5.0.pdf

**GEDCOM 5.3** exists in original form as a R82 (Word Perfect 5.1) and T82 (DOS text) documents.  
https://www.saintclair.org/ftp/pub/pafutils/geddoc.zip

The Word Perfect file can still be opened in current versions of Microsoft Word, retaining all original formatting.  For legibility the font was changed to Times before being converted to pdf:
https://chronoplexsoftware.com/gedcomvalidator/gedcom/gedcom-5.3.pdf

**GEDCOM 5.4** is only known to exist as an HTML conversion of the Word Perfect original.  
http://www.lege.com/gedcom54.html

This was manually reformatted in Microsoft Word and then converted to pdf:
https://chronoplexsoftware.com/gedcomvalidator/gedcom/gedcom-5.4.pdf

**Comment by chronoplexsoftware** [comment]
More over, we would be happy to contribute editable Microsoft Word versions of GEDCOM 3.0, 4.0, 5.0, 5.3 and 5.4 in order that all these older specifications can continue to exist in a modern, accessible, and readily editable format should any further transcription errors or conversion work be required.

**Comment by tychonievich** [comment]
Thanks for this context, and for your work in creating the PDFs! I was not personally involved in collecting or posting these PDFs, but I'll make sure those who were respond to this issue.

I'm a bit unclear about provenance and permissions here.

1. Are you giving gedcom.io permission to host your PDFs?
2. How would you like to be cited?

**Comment by lkessler** [comment]
The earlier specs were previously collected by Tamura Jones and have for many years been available here: 
[https://www.tamurajones.net/FamilySearchGEDCOMSpecifications.xhtml](https://www.tamurajones.net/FamilySearchGEDCOMSpecifications.xhtml)

Tamura's webpage includes a PDF of GEDCOM 3.0 which obtained by Brian Madsen.  Note that Tamura credits those who found or provided copies of each document. At the bottom of the page, Tamura includes links to some other relevant articles of his about the history of GEDCOM standards as well as 3 links to my blog posts when I rediscovered some some of the early standards (3.0, 4.0 and 5.0) 

Also see [this article by Tamura about GEDCOM 1.0](https://www.tamurajones.net/GEDCOM1.0.xhtml) as well as the links below it to my two blog posts, 

Also very relevant to history is the GEDCOM Specification, Future Direction Draft, by the Family History Department of The Church of Jesus Christ of Latter-day Saints, 7 July 1999, which Tamura has on his page.

And then there's the GEDCOM Event-Oriented Form, Draft Release 1.0, 12 Sept 1994, prepared by COMMSOFT which I don't believe is available online, but I have a copy of it.

**Comment by chronoplexsoftware** [comment]
>     Are you giving gedcom.io permission to host your PDFs?
Yes. Now that GEDCOM has an official home any efforts to preserve earlier specifications should be consolidated here.  Our main concern is ensuring that these older specifications can continue to exist in an accessible future proof format, and can be freely shared (note specifically the copyright notice in GEDCOM 3.0 and GEDCOM 4.0 which is more restrictive that in GEDCOM 5.x).

>     How would you like to be cited?
This issue is a sufficient record of the conversion works undertaken.

**Comment by chronoplexsoftware** [comment]
> And then there's the GEDCOM Event-Oriented Form, Draft Release 1.0, 12 Sept 1994, prepared by COMMSOFT which I don't believe is available online, but I have a copy of it.

It is available online in both pdf and WordPerfect formats:
http://www.saintclair.org/ftp/pub/pafutils/eged10ww.zip
http://www.saintclair.org/ftp/pub/pafutils/eged10wp.zip
